### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by mjeann**
Bug fix https://github.com/sap-tutorials/Tutorials/issues/7118

### CODE REVIEW

File: tutorials/cos-custom-rules/cos-custom-rules.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Create a Custom Goal with SAP Customer Order Sourcing
 description: Learn how to create a custom goal from start to finish with SAP Customer Order Sourcing.
 auto_validation: true
 time: 5
-tags: [ tutorial>beginner, topic>sap-api-business-hub]
+tags: [ tutorial>beginner, products>sap-business-technology-platform]
```
Comment (MichaelCzcz) [inline tutorials/cos-custom-rules/cos-custom-rules.md:6]: The name space "products" is no longer valid. Please use "software-product" instead.

**Comment by mjeann** [comment]
Changed products to software-product.
Also removed part of the tutorial, because or booster is not available in Trial anymore.

File: tutorials/cos-getting-started-trial/cos-getting-started-trial.md [hunk 2]
```diff
@@ -39,28 +40,9 @@ You are the owner of an e-commerce business or a web shop in the fashion industr
...
-
-You want to learn how to setup SAP Customer Order Sourcing yourself? This tutorial can help: [Set Up SAP Customer Order Sourcing Manually](cos-manual-setup).
-
-[DONE]
-[ACCORDION-END]
-
 [ACCORDION-BEGIN [Step 3: ](Create a new environment)]
 
-Having created a service instance and the corresponding credentials, you can now upload master data into SAP Customer Order Sourcing. You do this via a REST API.
+After creating a service instance and the corresponding credentials, for more information see [Set Up SAP Customer Order Sourcing Manually](cos-manual-setup), you can upload master data into **SAP Customer Order Sourcing**. You do this via a REST API.
```
Comment (MichaelCzcz) [inline tutorials/cos-getting-started-trial/cos-getting-started-trial.md:45]: Are you sure that you want this link to be in the middle of a sentence?

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for the updates. There is one final issue that I would like you to look into.

File: tutorials/cos-getting-started-trial/cos-getting-started-trial.md (hunk 2 above)
Comment (mjeann) [reply tutorials/cos-getting-started-trial/cos-getting-started-trial.md]: Changed it.

**Pull request merged**